tools/acrnlog/README.rst

```diff
@@ -57,6 +57,11 @@ output to the console. For example:
    ACRN:\>get_loglevel
    console_loglevel: 2, mem_loglevel: 5
 
+.. note::
```

Comment:
Rather than as an afterthought, it would be better to add this information earlier in the document.  Delete this addition and instead add it above on line 45:

    You can use ``get_loglevel`` and ``set_loglevel`` commands
    in the hypervisor shell (not the Service OS shell)
    to query and change the hypervisor log level.

and on line 50, just to emphasize it again, change the last sentence to:

    For example, in the hypervisor shell you can use these commands:

When we have a document to list all the shell commands we
can add a reference to that here too.

Reply:
Just force-pushed a new version. Thanks